src/lib/operators/sort.hpp

```diff
@@ -15,18 +15,27 @@
 namespace opossum {
 
 /**
- * Operator to sort a table by a single column. This implements a stable sort, i.e., rows that share the same value will
- * maintain their relative order.
- * Multi-column sort is not supported yet. For now, you will have to sort by the secondary criterion, then by the first
+ * Defines in which order a certain column should be sorted.
+ */
+struct SortColumnDefinition final {
```

Comment:
This is only a minor issue, but I think there might be a better name than _Definition_, e.g. `ColumnSortParameter`.

Reply:
Hmm actually I think _Definition_ is a better fit here since it is indeed defining how the column should be sorted... other opinions?